sklearn/metrics/_scorer.py

```diff
@@ -110,19 +117,28 @@ def __call__(self, estimator, *args, **kwargs):
         cache = {} if self._use_cache(estimator) else None
         cached_call = partial(_cached_call, cache)
 
+        if _routing_enabled():
+            routed_params = process_routing(self, "score", kwargs)
```

Comment:
Codecov says there is no coverage here. I'm guessing we need a test to check routing with `_MultimetricScorer`.